File: 9주차_기본정렬/복습/9주차_복습_박지윤/2750.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collections;
import java.util.Scanner;
public class Main {
        public static void main(String[] args) {
            Scanner n=new Scanner(System.in);
            int num=n.nextInt();
            
            ArrayList<Integer> arr=new ArrayList<>();

            for(int i=0;i<num;i++)
            {
                int input=n.nextInt();
                arr.add(input);
            }
            Collections.sort(arr);

            for(int i=0;i<num;i++){
                System.out.println(arr.get(i));
            }
        }
    }

File: 9주차_기본정렬/복습/9주차_복습_박부성/2750.py

n = int(input())
arr= []
for _ in range(n):
    arr.append(int(input()))

for i in range(n):
    tmp = i
    for j in range(n):
        if arr[tmp] < arr[j]:
            arr[tmp], arr[j] = arr[j], arr[tmp]

for num in arr:
    print(num)